lib/Controller/BanController.php

```diff
@@ -45,25 +47,39 @@ public function __construct(
 	#[PublicPage]
 	#[RequireModeratorParticipant]
 	public function banActor(string $actorType, string $actorId, string $internalNote = ''): DataResponse {
-		if ($actorId === 'wrong') {
+		try {
+			$attendee = $this->participant->getAttendee();
+			$roomId = $attendee->getRoomId();
+			$bannedByActorId = $attendee->getActorId();
+			$bannedByActorType = $attendee->getActorType();
+
+			$ban = $this->banService->createBan(
+				$actorId,
+				$actorType,
+				$roomId,
+				$bannedByActorId,
+				$bannedByActorType,
+				null,
+				$internalNote
+			);
+
+			return new DataResponse(
+				[
+					'id' => $ban->getId(),
+					'actorType' => $ban->getActorType(),
+					'actorId' => $ban->getActorId(),
+					'bannedType' => $ban->getBannedByActorType(),
+					'bannedId' => $ban->getBannedByActorId(),
+					'bannedTime' => $ban->getBannedAt()->getTimestamp(),
+					'internalNote' => $ban->getInternalNote(),
+				],
```

Comment:
You added `jsonSerialize()` to the Ban model. That function should return what you need here, if it does not you should adjust it.
Same in the list endpoint.